app/services/restream.ts

```diff
@@ -69,26 +69,32 @@ export class RestreamService extends StatefulService<IRestreamState> {
     });
   }
 
+  get views() {
+    class RestreamView extends ViewHandler<IRestreamState> {
+      /**
+       * This determines whether the user can enable restream
+       * Requirements:
+       * - Has prime, or
+       * - Has a grandfathered status enabled
+       */
+      get canEnableRestream() {
+        const userView = this.getServiceViews(UserService);
+        return userView.isPrime || (userView.auth && this.state.grandfathered);
+      }
+    }
```

Comment:
Can you explain why you're starting the pattern of defining the view handler inside the getter like this?

Reply:
@holiber bump

Reply:
I have only one method here and we don't export this class outside the file. So just don't see a big difference between declaring this class separately